test/widgets/test_thermal_zone.py

```diff
@@ -0,0 +1,16 @@
+import os
+from libqtile import widget
+
+
+def test_thermal_zone_getting_value():
+    # Create temporary zone file
+    tmp = '/var/tmp/qtile/test/widgets/thermal_zone'
+    zone_file = tmp + '/sys/class/thermal/thermal_zone0/temp'
+    os.makedirs(os.path.dirname(zone_file), exist_ok=True)
+
+    with open(zone_file, "w") as f:
+        f.write("22000")
+
+    thermal_zone = widget.ThermalZone(zone=zone_file)
+    output = thermal_zone.poll()
```

Comment:
This fails because `poll` tries to set `self.layout.colour` but `self.layout` is `None` as the widget has not been `_configure`d.

You could define a dummy class and set `widget.layout` to be that e.g.:
``` python
    class FakeLayout:
        pass

    with open(zone_file, "w") as f:
        f.write("22000")

    thermal_zone = widget.ThermalZone(zone=zone_file)
    thermal_zone.layout = FakeLayout()
```

Reply:
Done